documentation/asciidoc/topics/adding_backup_locations.adoc

```diff
@@ -0,0 +1,67 @@
+[id='adding_backup_locations-{context}']
+= Adding Backup Locations to Caches
+When you configure {brandname} clusters to perform cross-site replication, you
+can add backup locations to your cache configurations.
+
+.Procedure
+
+. Create cache configurations with identical names for each site.
++
+Cache configurations at each site can use different cache modes and backup
+strategies. {brandname} replicates data based on cache names.
++
+. Configure sites to go offline automatically with the `take-offline` element.
+.. Set a value of `0` or `-1` for the `after-failures` attribute.
+.. Configure a minimum wait time of 10 seconds (10000 milliseconds).
```

Comment:
why 10 seconds? 
I'm not sure about this. In quicklabs, it took almost 1 min to promote a pod to site master. With 10 seconds, the cross-site replication will be disabled every time the site master pod dies.

Reply:
@Crumby can you advise here please?

Reply:
resolving this as I've discussed with Pavel over zulip